eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/parameters/InputParameters.java

```diff
@@ -118,4 +113,26 @@ public int getChunkSize() {
     public void setChunkSize(int chunkSize) {
         this.chunkSize = chunkSize;
     }
+
+    public JobParameters toJobParameters() {
+        return new JobParametersBuilder()
+                .addString("projectAccession", projectAccession)
```

Comment:
This is a duplicate, it was added before in a cluster of xxxAccession parameters.

Reply:
I don't see where is projectAccession being added to the JobParameters. Is in a separate PR?

Reply:
See line 126, it was added [in a previous commit](https://github.com/pabarcgar/eva-accession/commit/a4f1123298a62be9ec78cc6751f05d6b100480bb#diff-eb880f5135a6a435c483e4f9f6b6b312) on this same PR :joy:

Reply:
OMG! It was in front of me!